go/database/mpt/verification_test.go

```diff
@@ -41,9 +43,9 @@ var forestFiles = []string{
 	"values/values.dat",
 }
 
-func TestVerification_VerifyValidForest(t *testing.T) {
+func TestVerification_VerifyValidMptState(t *testing.T) {
```

Comment:
here and below it seems many tests were changed to test one layer above the forest, but I think it is an unrelated change, because it did not change what is tested.

Reply:
I do not think this was resolved, there is still many tests that got changed, but it seems unrelated.